examples/DVCLive-YOLO.ipynb

```diff
@@ -1,5 +1,12 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"https://colab.research.google.com/github/iterative/dvclive/blob/main/examples/DVCLive-YOLO.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+   ]
+  },
```

Comment:
Is this supposed to be here?

Reply:
Yes, if you go to the notebook in GitHub UI or local, you could click the button. Including the cell appears to be common for that

Reply:
oh, yes, sorry I also added to YOLO because I forgot to add it in its P.R.